DocumentationAnalyzers/DocumentationAnalyzers.Test/RefactoringRules/DOC900UnitTests.cs

```diff
@@ -225,6 +225,30 @@ class TestClass<T> {
             await VerifyCodeFixAsync(testCode, fixedCode);
         }
 
+        [Fact]
+        [WorkItem(42, "https://github.com/DotNetAnalyzers/DocumentationAnalyzers/issues/42")]
+        public async Task TestPropertyDocumentationWithCodeBlockAsync()
+        {
+            var testCode = @"
+class TestClass {
+    ///$$ <summary>
+    /// Provide a `value`.
+    /// </summary>
+    int Method => 3;
```

Comment:
`int Property => 3;` or more copy paste friendly `int Foo => 0;`

Reply:
I use the longer names to avoid unnecessary squiggles from the spell checker, but yes I definitely meant `Property` here.

Reply:
What spellchecker do you use? With respeller it is easy to add custom words to team shared dictionary.

Reply:
`int Member=> 0;` is copy paste friendly.

Reply:
I use Visual Studio Spell Checker. It also allows words to be added, but I've avoided adding 'foo' due to dotnet/roslyn#21224.

Reply:
➡️ Updated property name in 9c083abda8c36639625e99db12b27112f933d0a8